content/blog/2024-05-22-rustl-2024-unconference.md

```diff
@@ -0,0 +1,196 @@
+We listened to talks, Raph gave one, Rik Arends did the "hot reloading my entire editor live" thing that makes every other Rust GUI developer jealous, it was an all-around good time.
+
+(Videos for individual talks aren't up yet, but there are livestream recordings of [Day 1](https://www.youtube.com/watch?v=XLefuzE-ABU) and [Day 2](https://www.youtube.com/watch?v=521NfGf7AR0) on Youtube.)
+
+After RustNL, though, another less-documented event called [The Unconference](https://2024.rustnl.org/unconf/). It was from roughly the same organizers and took place in the same city, but the format was fairly different.
+
+The Unconference wasn't livestreamed, and I haven't found any public discussion of it, so I thought I'd take the time to describe a bit of what was going.
+
+
+## The format
+
+The "Unconference" was two days of loosely-structured chats between members of various Rust project teams where people talked about ecosystem collaboration, complained to Lang Team members that their favourite feature wasn't implemented yet, [went to pester everyone else about variadic generics](https://poignardazur.github.io/2024/05/25/report-on-rustnl-variadics/), and other good fun.
+
+Attendees were split into three teams:
+
+- The Rust project.
+- GUI and Applications.
+- Embedded development.
+
+Discussions inside of these teams was mostly self-organized, with cross-team discussion mostly taking place during lunch and recesses.
+
+
+### On the tyranny of struturelessness
+
+*Note: This section is very much my personal opinion and doesn't represent the Linebender project as a whole.*
+
+I'm not sure the self-organized format worked well.
+
+From what I saw, the Embedded team took well to it, and the Rust team was kept productive thanks to Alice Cecile's efforts in marshalling everyone.
+
+In the case of the GUI team, people were spread in a very large room, which should have been conducive to small side-discussions and people splitting up to talk about the problems that interested them. Instead, there was an unspoken accord to progress through agenda items one by one, with a few people dominating the discussions on these items.
+
+Because those people we spread around a large room, they had to talk loudly to address each other, which left little room for side-discussions.
+Because the people talking were naturally the more confident and extraverted, more introverted people ended up taking a passive role in the discussion.
+
+To me, this feels like a strategic mistake.
+It was a setup that encouraged bikeshedding and long back-and-forths, and discouraged plurality of opinions.
+It's no coincidence that the most interesting conversations of the Unconference happened at lunch: lunch was the point of the Unconference where people were most mixed, had the most spontaneous conversations, and were least constrained by having to follow what someone else was saying.
+
+Our discussion were still productive, I just feel like the format could have been improved.
```

Comment:
thanks for the critiques and feedback here; we'll definitely incorporate this and the other feedback we received for next time. 

Regarding the agenda/topics/structure, given the total absence of replies to my requests before the unconf, I felt like I had no alternative but to bring up the suggested topical agenda at the top of the actual unconf proceedings. Hopefully, with more time to plan ahead of future unconfs (and ideally some additional participation from UI experts in the planning phase), we can receive more input from the attendees and come up with an agreed-upon agenda before we actually meet.

Reply:
The web engines hackfest had people suggest breakout sessions as github issues (https://github.com/Igalia/webengineshackfest/issues). These were then scheduled prior to the start of the conference (which some room for reorganisation) and people could choose which sessions they wanted to attend. I think this is quite a good way of doing it as it allows everyone to participate in negotiating session content rather than putting everything on a central organiser. I think your bound to not get much in the way of responses if people are asked to submit privately.

Not blaming you personally - it's a learning experience for everyone - and you at least stepped up to try and organise things. But some suggestions for next time :)

Reply:
> suggest breakout sessions as github issues

that's a good idea!


> you at least stepped up to try and organise things

i was "voluntold", haha, but yea, someone's gotta do it